frontend/src/components/OperationDataTable/OperationDataTable.tsx

```diff
@@ -45,25 +36,11 @@ const renderTableRows = (data: List<OperationDataMap>, columns: (string | number
 };
 
 export const OperationDataTable: React.SFC<OperationDataTableProps> = (props) => {
-  const { fetchData, id, limit, offset, list } = props;
-  const columns = props.list ? getColumns(props.list.get(0)) : [];
+  const { limit, offset, list, prev, next, first, hidePaginate } = props;
+  const columns = props.columns ? props.columns : [];
   const max = offset + limit;
 
-  const goToFirst = () => fetchData({ id, limit, offset: 0 });
-  const goToNext = () => {
-    const newOffset = offset + limit;
-    if (list && list.count() >= limit) {
-      fetchData({ id, limit, offset: newOffset });
-    }
-  };
-  const goToPrev = () => {
-    if (offset > 0) {
-      const newOffset = offset - limit;
-      fetchData({ id, limit, offset: newOffset });
-    }
-  };
-
-  if (list && list.count() && props.columns) {
+  if (list && list.count && props.columns) {
```

Comment:
Unless your list isn't of type `List`, count is still a function that needs to be called too return an actual count ... i.e `list.count()`... otherwise, it'll always be true, even with an empty list.